KsApi/models/templates/CommentTemplates.swift

```diff
@@ -46,10 +46,27 @@ extension Comment {
     id: "89DJa89jdSDJ89sd8==",
     isDeleted: false,
     parentId: "Q29tbWVudC0zMjY2NDAxMg==",
-    replyCount: 2,
+    replyCount: 0,
```

Comment:
Ah good catch. Replies always return a 0 for this count, right?

Reply:
Yup, none of the tests broke either after the change, which is good because that would indicate we were checking replies on replies.